### DISCUSSION TITLE
How do I install an older GPTK version into Whisky?

### DISCUSSION THREAD

**Discussion by serista** [Whisky-App/Whisky#491] [Q&A]
Hi, the title says it all. 
F.e. if performance/compatibility of a newer version seems worse or I need to compare different versions performance.

More precisely, I want to return to GPTK beta 4 after installing 1.0.

**Comment by mavethee** [comment]
Once Whisky is highlighted,
Whisky -> Setup -> GPTK -> Uninstall.
Install manually from DMG desired version.

**Reply by serista** [reply]
> Install manually from DMG desired version.

How exactly do I do that? Once I uninstall GPTK there are only two buttons: quit/exit and next/continue. After pressing Next it starts to download the newest GPTK version again and Quit closes the program.

**Reply by nikita-fuchs** [reply]
Same question here, how does one find an installer for old versions ? Since updating to the newest one, Dying LIght 2 doesn't start anymore.
